styles/carousel.css

```diff
@@ -0,0 +1,146 @@
+
+.cards-container {
+  display: flex;
+  padding: 0 72px;
+  background-color: #fff;
+  box-shadow: none;
+ }
+
+ .cards-container > .default-content-wrapper {
+  flex: 1;
+  padding: 20px;
+ }
+
+ .cards-container > .default-content-wrapper > div:first-child {
```

Comment:
cards is a different block, which use class name .cards
Please use other class name

Reply:
Done